cadence/contracts/DapperStorageRent.cdc

```diff
@@ -0,0 +1,247 @@
+import FungibleToken from "./FungibleToken.cdc"
+import FlowToken from "./FlowToken.cdc"
+import PrivateReceiverForwarder from "./PrivateReceiverForwarder.cdc"
+
+/// DapperStorageRent
+/// Provide a means for accounts storage TopUps. To be used during transaction execution.
+pub contract DapperStorageRent {
```

Comment:
Is it essential to have the prefix of the name `Dapper`?

Reply:
No,  not essential, but was thinking it gave some good context to its relationship to [BloctoStorageRent](https://flowscan.org/contract/A.1dfd1e5b87b847dc.BloctoStorageRent)

Reply:
Hmm.. I think it is better to remove and not use the company name as the prefix. Instead, we can go with a generic contract name to avoid some "XYZ" reasons. We can rename it to  `Refueler`  or something like that.